docs/fundamentals/networking/quic/quic-overview.md

```diff
@@ -30,7 +30,36 @@ The QUIC implementation was introduced in .NET 5 as the `System.Net.Quic` librar
 > [!NOTE]
 > In .NET 7.0, the APIs are published as [preview features](https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md).
 
-From the implementation perspective, `System.Net.Quic` depends on [MsQuic](https://github.com/microsoft/msquic), the native implementation of QUIC protocol. As a result, `System.Net.Quic` platform support and dependencies are inherited from `MsQuic` and documented in [HTTP/3 Platform dependencies](../../../core/extensions/httpclient-http3.md#platform-dependencies). In short, the `MsQuic` library is shipped as part of .NET for Windows. But for Linux, `libmsquic` must be manually installed via an appropriate package manager. For the other platforms, it's still possible to build `MsQuic` manually, whether against SChannel or OpenSSL, and use it with `System.Net.Quic`. However, these scenarios are not part of our testing matrix and unforeseen problems might occur.
+From the implementation perspective, `System.Net.Quic` depends on [MsQuic](https://github.com/microsoft/msquic), the native implementation of QUIC protocol. As a result, `System.Net.Quic` platform support and dependencies are inherited from MsQuic and documented bellow in [Platform dependencies](#platform-dependencies). In short, the MsQuic library is shipped as part of .NET for Windows. But for Linux, `libmsquic` must be manually installed via an appropriate package manager. For the other platforms, it's still possible to build MsQuic manually, whether against SChannel or OpenSSL, and use it with `System.Net.Quic`. However, these scenarios are not part of our testing matrix and unforeseen problems might occur.
+
+## Platform dependencies
+
+### Windows
+
+- Windows 11, Windows Server 2022, or later. (Earlier Windows versions are missing the cryptographic APIs required to support QUIC.)
+
+On Windows, msquic.dll is distributed as part of the .NET runtime, and no other steps are required to install it.
+
+### Linux
+
+All the following dependencies are stated in `libmsquic` package manifest and will be automatically installed by the package manager:
+
+- OpenSSL 3+ or 1.1 - depends on the default OpenSSL version for the distribution version, e.g. OpenSSL 3 for Debian 12 and OpenSSL 1.1 for Debian 11
```

Comment:
```suggestion
- OpenSSL 3+ or 1.1 - depends on the default OpenSSL version for the distribution version, for example, OpenSSL 3 for Debian 12 and OpenSSL 1.1 for Debian 11.
```

Reply:
I think we should mention Ubuntu 22+ for OpenSSL 3 as well

Reply:
I'm not opposed to this. But one thing that slightly bothers me is that this is not supposed to be an exhaustive list and with more than one example it might look like it is.
And we know it's not just Debian and Ubuntu, it's Fedora, OpenSUSE, RHEL derivatives ...

So do you still think we should explicitly name Ubuntu here as well? If so, I'll add it.

Reply:
Or I can swat Debian for Ubuntu example?

Reply:
I don't know how common these distros are, so if you think Debian is the more common one then you can keep it as-is.

Reply:
I swapped it for Ubuntu.